server/app/playbook_run.go

```diff
@@ -427,6 +427,17 @@ type AssignedTask struct {
 	ChecklistItem
 }
 
+// RunAction represents the run action settings. Frontend passes this struct to update settings.
+type RunAction struct {
+	BroadcastChannelIDs       []string `json:"broadcast_channel_ids"`
+	Followers                 []string `json:"followers"`
+	WebhookOnStatusUpdateURLs []string `json:"webhook_on_status_update_urls"`
+
+	StatusUpdateBroadcastChannelsEnabled  bool `json:"status_update_broadcast_channels_enabled"`
+	StatusUpdateBroadcastFollowersEnabled bool `json:"status_update_broadcast_followers_enabled"`
+	StatusUpdateBroadcastWebhooksEnabled  bool `json:"status_update_broadcast_webhooks_enabled"`
```

Comment:
Should we remove now the followers-related fields in the struct now?
```suggestion
	BroadcastChannelIDs       []string `json:"broadcast_channel_ids"`
	WebhookOnStatusUpdateURLs []string `json:"webhook_on_status_update_urls"`

	StatusUpdateBroadcastChannelsEnabled  bool `json:"status_update_broadcast_channels_enabled"`
	StatusUpdateBroadcastWebhooksEnabled  bool `json:"status_update_broadcast_webhooks_enabled"`
```

Reply:
I've created a separate branch to remove the followers-related fields. Do you think it's better if I include those changes in this PR?

Reply:
@gbochora, I'm ok removing the column from the migration in another PR, but I think it's better if we don't add the `Followers` and `StatusUpdateBroadcastFollowersEnabled` to this new struct in this PR.

Reply:
Agree. I've removed the followers-related code, you can review just the last commit. Thanks.